appveyor.yml

```diff
@@ -68,7 +68,7 @@ build_script:
       --unmanaged
       %GCLIENT_EXTRA_ARGS%
       "https://github.com/electron/electron"
-  - gclient sync --with_branch_heads --with_tags --reset
+  - gclient sync --with_branch_heads --with_tags --reset --break_repo_locks
```

Comment:
This doesn't seem like it should be necessary... we get a new VM every time right? Also, I've seen issues with this flag not doing what it says on my machine.

What precipitated this change?

Reply:
@nornagon it was an attempt to resolve this issue, but it didn't help:
LockError: C:\Users\electron\libcc_cache\chromium.googlesource.com-external-github.com-google-googletest is already locked
From:
https://ci.appveyor.com/project/electron-bot/electron-ia32-testing/builds/26210208

I can spin up a VM and debug further.

Reply:
`--ignore_repo_locks` works better than break, iirc. would be good to know how it managed to get locked though.

Reply:
I just looked at a copy of the master VM.  There were a couple of lock files there.  I prewarmed the cache and I'm guessing somehow Ieft locks.  I have cleaned up the master image so that there aren't any locks on it, so next run we should be good.